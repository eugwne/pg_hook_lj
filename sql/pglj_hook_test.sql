show shared_preload_libraries;
create user badpass with password 'bad';
create user badpass with password 'better';
drop user badpass;
