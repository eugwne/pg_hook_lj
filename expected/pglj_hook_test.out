show shared_preload_libraries;
 shared_preload_libraries 
--------------------------
 pg_hook_lua
(1 row)

create user badpass with password 'bad';
ERROR:  password is too short
create user badpass with password 'better';
drop user badpass;
